integration-cli/docker_cli_build_test.go

```diff
@@ -5444,3 +5444,30 @@ func (s *DockerTrustSuite) TestBuildContextDirIsSymlink(c *check.C) {
 		c.Fatalf("build failed with exit status %d: %s", exitStatus, out)
 	}
 }
+
+// Issue #15634: COPY fails when path starts with "null"
+func (s *DockerSuite) TestBuildNullStringInAddCopyVolume(c *check.C) {
+	name := "testbuildnullstringinaddcopyvolume"
+
+	ctx, err := fakeContext(`
+		FROM busybox
+		
+		ADD null /
+		COPY nullfile /
+		VOLUME nullvolume
+		`,
+		map[string]string{
+			"null":     "test1",
+			"nullfile": "test2",
+		},
+	)
+
+	if err != nil {
```

Comment:
You can replace this if-statement with: `c.Assert(err, check.IsNil)`  to be consistent with the other test clean-up PRs.